site/public/js/calendar.js

```diff
@@ -125,28 +125,65 @@ function darken(colorstr) {
     }
 }
 
+/**
+ * This function returns a slightly lighter color than the color variable name passed.
+ *
+ * @param colorstr : string the color to lighten in the form "var(--color-name)"
+ * @returns {string} a hex code for a slightly lighter shade
+ */
+function lighten(colorstr) {
```

Comment:
This function does not always return a valid hex code.
For example, `#1B5E20 => #3D7G42`
So for some course colors, the background color won't display.